src/components/containers/notification/Notification.js

```diff
@@ -146,6 +146,13 @@ function renderCallsToAction(callsToAction) {
       {callsToAction.map((callToAction, index) => {
         const buttonStyleType = index === 0 ? 'primary' : 'default';
 
+        if (React.isValidElement(callToAction)) {
```

Comment:
I think this is a bit complex for a notification component.  There's nothing preventing someone from putting whatever other buttons or content they want into the notification via children to begin with, and I actually like that.

I agree with Jake (and let me know if I'm not understanding your point correctly) that the CTAs could be pulled out and put in a more complex component in ``Patterns``.